app.js

```diff
@@ -1,10 +1,12 @@
 const { getTopics, getEndPoints, getArticlesById  } = require("./controller");
-const express = require('express')
+const express = require('express');
+const { invalidEndPoint } = require("./error-handling");
 const app = express();
 
 app.get('/api/topics', getTopics)
 app.get('/api', getEndPoints)
 app.get('/api/articles/:article_id',getArticlesById)
+app.use(invalidEndPoint)
```

Comment:
the error handlers should be split by functionality (eg handleCustomErrors, handlePsqlErrors) instead of invalidEndPoint

Reply:
Updated this and changed name to handleCustomErrors